pkg/utils/file/download.go

```diff
@@ -114,6 +114,16 @@ func getCachePath(cacheDir, src string) (string, error) {
 	case "http", "https":
 		return path.Join(cacheDir, u.Scheme, u.Host, u.Path), nil
 	default:
+		if _, err := os.Stat(src); err != nil {
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/4cc651b6334ea0d3e8c484e0a47ca6186bc5686e/pkg/utils/path/path.go#L23-L41

We can add the function `Expand`, which expands the home and local paths, and then replaces function `ExpandHome`, use it here.

Reply:
Nice idea, can I add the check file to it?

Reply:
According to the function definition of this piece, it seems that there is no need to check whether the file exists.

Reply:
> According to the function definition of this piece, it seems that there is no need to check whether the file exists.

Yep